internal/v1/handler_blueprints_test.go

```diff
@@ -24,7 +24,7 @@ func TestHandlers_CreateBlueprint(t *testing.T) {
 	dbase, err := dbc.NewDB()
 	require.NoError(t, err)
 
-	db_srv, tokenSrv := startServer(t, "", "", &ServerConfig{
+	db_srv, tokenSrv := startServer(t, &testServerClientsConf{}, &ServerConfig{
```

Comment:
I was gonna say I don’t like `startServer` it is going off-hands with another client being added in other PR BUT you actually make it better by introducing a struct that is easier on eyes as things can be skipped. Nice. @mgold1234 you will need to rebase on top of this.